.github/workflows/site.yaml

```
@@ -11,8 +11,8 @@ jobs:
     name: ${{ matrix.config.os }} (${{ matrix.config.r }})
 
     strategy:
-      # we keep a matrix for convenience, but we would always want to run on one
-      # single OS and R version
+      # we keep a matrix for convenience, but we would typically just run on one
+      # single OS and R version, aligned with the target deployment environment
```

This should be reverted, there is no target deployment environment for the website. This was the adaptation we did for the Shiny CI/CD workflows, which does not apply back to the techguides though.
```suggestion
      # we keep a matrix for convenience, but we would always want to run on one
      # single OS and R version
```